src/app/checkout/payment/payment-modes-list/payment-modes-list.component.html

```diff
@@ -5,10 +5,41 @@
 </div>
 
 <div class="selected-mode" [ngSwitch]="selectedMode.name">
-  <ng-template [ngSwitchCase]="'Check'">
-    <app-cash-on-delivery (payOnDelivery)="makePayment()"></app-cash-on-delivery>
+  <ng-template [ngSwitchCase]="'COD'">
+    <app-cash-on-delivery (payOnDelivery)="makePaymentCod()"></app-cash-on-delivery>
   </ng-template>
   <ng-template [ngSwitchCase]="'Payubiz'">
```

Comment:
Why string comparison. Also if payment mode name is changed, then what!! redeployment ???